socrates/test/members/members_test.js

```diff
@@ -123,12 +135,77 @@ describe('SoCraTes members application', function () {
     });
 
     it('allows a Softwerkskammer member to edit his page', function (done) {
+      sinon.stub(subscriberstore, 'getSubscriber', function (nickname, callback) { callback(null, softwerkskammerSubscriber); });
+
       appWithSoftwerkskammerMember
         .get('/edit')
         .expect(200)
         .expect(/Here you find the information from your Softwerkskammer account that is used by SoCraTes\./, done);
     });
 
+    it('does not allow an unsubscribed member to enter a roommate', function (done) {
```

Comment:
Und "done" ist auch Parameter? Und dann mache ich für jeden Fall einen Test, in dem nur die Funktion aufgerufen wird? Ist es das, was Du im Sinn hast?